generator/osm_element.cpp

```diff
@@ -28,6 +28,7 @@ std::string DebugPrint(OsmElement::EntityType e)
     case OsmElement::EntityType::Member:
       return "member";
   }
+  return "unknown";
```

Comment:
Please add an assertion here. Something like
```
auto const eInt = static_cast<size_t>(OsmElement::EntityType);
string const msg = "Unknown OsmElement::EntityType " + strings::toString(eInt);
ASSERT(false, (msg));
return msg;
```

Reply:
Going by the name this is just a "DebugPrint" function, does it really make sense to clutter it with such a verbose assert?
My patch was fairly incomplete (I'll try to upload the more complete version later), if you'd want to have asserts for all such cases that would result in significant code bloat I think.

Reply:
Why not if the goal is to catch incorrect usage attempts.
See the discussion below, looks like adding CHECKs is the preferred way.